### PR TITLE
Removed duplicated recording status.

### PR DESCRIPTION
**Pull request by mateuszglapiak-dolby**

### CODE REVIEW

File: lib/src/sdk_api/models/recording.dart [hunk 1]
```diff
@@ -35,6 +35,17 @@ enum RecordingStatus {
...
+  static RecordingStatus valueOf(String? value) {
+    final lowerCaseValue = value?.toLowerCase();
+    return RecordingStatus.values.firstWhere(
+      (element) {
+        return element._value == value ||
+            element.name.toLowerCase() == lowerCaseValue;
+      },
+      orElse: () => throw Exception("Invalid enum name"),
+    );
+  }
```
Comment (graduad) [inline lib/src/sdk_api/models/recording.dart:47]: Do we need a `valueOf` and a `decode` method?

**Review by graduad — COMMENTED** [review]
Apart from my comment it looks good.

**Review by graduad — COMMENTED** [review]
Apart from my comment it looks good.

File: lib/src/sdk_api/models/recording.dart [hunk 2]
```diff
@@ -35,9 +32,15 @@ enum RecordingStatus {
...
-        .firstWhereOrNull((element) => element._value == value);
+  static RecordingStatus decode(String? value) {
+    final lowerCaseValue = value?.toLowerCase();
+    return RecordingStatus.values.firstWhere(
+      (element) {
+        return element._value == value ||
+            element.name.toLowerCase() == lowerCaseValue;
+      },
+      orElse: () => throw Exception("Invalid enum name"),
+    );
```
[the remark this answers is not in the document]
Comment (mateuszglapiak-dolby) [reply lib/src/sdk_api/models/recording.dart:43]: I'll revert the optional decode. We will need to check `RecordingStatusUpdate` event which requires non-optional. Maybe we will need to put default value there?

**Review by graduad — APPROVED** [review]

**Pull request merged**